### ISSUE TITLE
Similarity in Context example from website not working as expected

### ISSUE THREAD

**Issue by mlocutus** [explosion/spaCy#1791]
Running the similarity example from the website in Spacy 2.0.5 produces constant similarity scores rather than the expected values as shown from the example.

https://spacy.io/usage/vectors-similarity#in-context

Here's the example:

```
import spacy
nlp = spacy.load('en_vectors_web_lg')

doc1 = nlp(u"The labrador barked.")
doc2 = nlp(u"The labrador swam.")
doc3 = nlp(u"the labrador people live in canada.")

for doc in [doc1, doc2, doc3]:
    labrador = doc[1]
    dog = nlp(u"dog")
    print(labrador.similarity(dog))
```

I've tried both the `en_core_web_lg` and `en_vectors_web_lg` models, but receive constant similarity score values either way.

the website indicates we should see scores of `0.56`, `0.48` and `0.39` but I always get:

```
0.634902616248
0.634902616248
0.634902616248
```

## Your Environment
* **Python version:** 3.5.2
* **Platform:** Linux-4.8.0-56-generic-x86_64-with-Ubuntu-16.04-xenial
* **Models:** en_vectors_web_lg, en_core_web_lg
* **spaCy version:** 2.0.5

**Comment by zumansky** [comment]
You should be able to make it work by changing the model to 'en_core_web_sm' or 'en'.
The example on the website shows how Spacy can guess the meaning of a word that hasn't been seen before based on the context. Therefore, the word labrador from the example above has to be out of vocabulary.

**Comment by nscozzaro** [comment]
@zumansky, I noticed that if I make the word instead  an out-of-vocabulary word like labradorrr (I've also tried other out-of-vocab "words") it just yields 0:
`# nlp = spacy.load('en_core_web_lg')
doc1 = nlp(u"The labradorrr barked.")
doc2 = nlp(u"The labradorrr swam.")
doc3 = nlp(u"the labradorrr people live in canada.")

for doc in [doc1, doc2, doc3]:
    labrador = doc[1]
    dog = nlp(u"dog")
    print(labrador.similarity(dog))
>>> 0
>>> 0
>>> 0
`

Any ideas for resolving this? @mlocutus did you find a solution?

**Comment by cmxela** [comment]
Hi, Any comment on this issue, I'm facing the same problem. Is it a problem with the model or with the Spacy code? Does it has any sense to try to train a custome model?

**Comment by TomLaMantia** [comment]
I can confirm I am still having this issue with the latest version of Spacy running under Docker.

However, as @zumansky points out, the vectors for `labrador` differ across `doc1`, `doc2`, `doc3` when compared to `dog` if the word is OOV (`en_core_web_sm`). This makes sense, though the documentation may want to indicate that this feature is intended for words that the model is inferring.

**Comment by ines** [comment]
Sorry about this – with the other changes and improvements coming in #2274, we've now decided to take this example out, because it's too misleading. The "context" here comes from the context-sensitive tensors that are available in the `sm` models, which don't come with word vectors. When we released the models for v2.0, we thought this might be cool – but the similarity in context feature turned out to be much less useful.
